mayhem/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM --platform=linux/amd64 ubuntu:22.04 as builder
```

Comment:
Any reason there are two Dockerfiles and Mayhemfile copies?

Reply:
Weird, I don't know why this is happening. I think something went wrong when deleting and recreating my fork to use the mayhemheroes fork as upstream.